src/core/json-schema-components.jsx

```diff
@@ -85,7 +85,7 @@ export class JsonSchema_string extends Component {
       const Select = getComponent("Select")
       return (<Select className={ errors.length ? "invalid" : ""}
                       title={ errors.length ? errors : ""}
-                      allowedValues={ enumValue }
+                      allowedValues={ Array(...enumValue) }
```

Comment:
prefer spread array shorthand
```suggestion
                      allowedValues={ [...enumValue] }
```

Reply:
Got it, thanks.